scripts/approve_draft_registrations.py

```diff
@@ -0,0 +1,33 @@
+import sys
```

Comment:
Thanks for the notes on this PR, which helped me realize that this script (and certain others) were for testing/ local development use only.

Can you add a docstring to the top of this script, so that future developers can figure out what it is for? Ditto for other developer-only scripts in this PR.

(the scripts folder is getting very cluttered, and it's sometimes difficult to wade through and find a particular thing)